package.json

```diff
@@ -2,25 +2,43 @@
   "name": "eslint-visitor-keys",
   "version": "2.1.0",
   "description": "Constants and utilities about visitor keys to traverse AST.",
-  "main": "lib/index.js",
+  "type": "module",
+  "main": "dist/eslint-visitor-keys.cjs",
+  "exports": {
+    ".": [
+      {
+        "import": "./lib/index.js",
+        "require": "./dist/eslint-visitor-keys.cjs",
+        "default": "./dist/eslint-visitor-keys.cjs"
```

Comment:
I'm pretty sure we don't need both "default" and "require" here, one of them should suffice.

Reply:
I went ahead and removed, but note espree is still using it (the reason why I added it previously). I imagine a bundler could conceivably treat it differently, though Rollup is always satisfied if at least `import` or `require` is present. I don't see from https://webpack.js.org/guides/package-exports/ whether Webpack does so, but I would suspect it should work.

This leaves:

```json
"exports": {
    ".": [
      {
        "import": "./lib/index.js",
        "require": "./dist/eslint-visitor-keys.cjs"
      },
      "./dist/eslint-visitor-keys.cjs"
    ],
    "./package.json": "./package.json"
  },
```

...though I think we could also forego the fallback array if you want.

Reply:
Ah, I missed that Espree was using it. I'm pretty sure "default" and the fallback array are redundant with "require" in all relevant cases, but I'm fine with leaving this as-is.